tests/utils.py

```diff
@@ -475,7 +476,17 @@ class DummyWriter(DummyWriterMixin, AgentWriter):
     def __init__(self, *args, **kwargs):
         # original call
         if len(args) == 0 and "agent_url" not in kwargs:
-            kwargs["agent_url"] = "http://localhost:8126"
+            kwargs["agent_url"] = agent.get_trace_url()
+        if "api_version" not in kwargs:
+            kwargs["api_version"] = "v0.5"
+
+        self._trace_flush_enabled = False
+
+        # this is expected to be enabled as long as an agent is running and
+        # this instance was created by a DummyTracer
+        if kwargs.get("trace_flush_enabled", False) is True:
```

Comment:
This writing is a bit confusing, are we trying to verify that the parameter is supplied and is a bool?  If not, this should be `kwargs.get...` by itself.  If so, then you can use isinstance(...) to check.